sdk/mobile-center-updates/src/main/res/values/strings.xml

```diff
@@ -7,4 +7,7 @@
     <string name="mobile_center_updates_update_dialog_ignore">Ignore</string>
     <string name="mobile_center_updates_update_dialog_download">Download</string>
     <string name="mobile_center_updates_update_dialog_postpone">Postpone</string>
+    <string name="mobile_center_updates_dialog_actioned_on_disabled_toast">Updates were disabled</string>
```

Comment:
You mean `are`?

Reason for past tense is in case it's re-enabled again in the mean time, buttons do nothing after the first state change even if you re-enable.